conf/config-default.yaml

```diff
@@ -497,35 +510,36 @@ plugins:                          # plugin list (sorted by priority)
   - ext-plugin-post-req            # priority: -3000
   - ext-plugin-post-resp           # priority: -4000
 
-stream_plugins: # sorted by priority
+stream_plugins:                    # stream plugin list (sorted by priority)
   - ip-restriction                 # priority: 3000
   - limit-conn                     # priority: 1003
   - mqtt-proxy                     # priority: 1000
   #- prometheus                    # priority: 500
   - syslog                         # priority: 401
   # <- recommend to use priority (0, 100) for your custom plugins
 
-#wasm:
-  #plugins:
-    #- name: wasm_log
-      #priority: 7999
-      #file: t/wasm/log/main.go.wasm
-
-#xrpc:
-  #protocols:
-    #- name: pingpong
-
-plugin_attr:
-  log-rotate:
-    interval: 3600    # rotate interval (unit: second)
-    max_kept: 168     # max number of log files will be kept
-    max_size: -1      # max size bytes of log files to be rotated, size check would be skipped with a value less than 0
-    enable_compression: false    # enable log file compression(gzip) or not, default false
-  skywalking:
-    service_name: APISIX
-    service_instance_name: APISIX Instance Name
-    endpoint_addr: http://127.0.0.1:12800
-  opentelemetry:
+# wasm:
+#   plugins:
+#     - name: wasm_log
+#       priority: 7999
+#       file: t/wasm/log/main.go.wasm
+
+# xrpc:
+#   protocols:
+#     - name: pingpong
+
+plugin_attr:          # Plugin attributes
+  log-rotate:         # Plugin: log-rotate
+    interval: 3600    # Set the log rotate interval in seconds.
+    max_kept: 168     # Set the maximum number of log files to keep. If exceeded, historic logs are deleted.
+    max_size: -1      # Set the maximum size of log files in bytes before a rotation. 
+                      # Skip size check if max_size is less than 0.
+    enable_compression: false    # Compress (gzip) log files. Require tar to be available.
+  skywalking:                                     # Plugin: skywalking
+    service_name: APISIX                          # Set the service name for SkyWalking reporter.
+    service_instance_name: APISIX Instance Name   # Set the service instance name for SkyWalking reporter.
+    endpoint_addr: http://127.0.0.1:12800         # Set the SkyWalking HTTP endpoint.
```

Comment:
Need to merge https://github.com/apache/apisix/pull/9662 if `report_interval` indeed needs to be added